data/mods/MindOverMatter/powers/teleportation_eoc.json

```diff
@@ -100,6 +100,676 @@
       }
     ]
   },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_TELEPORTER_GATEWAY_MESSAGE",
+    "effect": [
+      {
+        "u_message": "There is a long moment of darkness and terrible cold that seems like it will freeze your heart, and then you are somewhere else.",
+        "type": "neutral"
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_TELEPORTER_ATTUNING_MESSAGE",
+    "effect": [
+      { "u_message": "You concentrate and begin memorizing every detail of the current area you are in.", "type": "neutral" }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_TELEPORTER_ATTUNED_MESSAGE",
+    "effect": [
+      {
+        "u_message": "You have successfully memorized the local area, and may now return to it with your powers.",
+        "type": "good"
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_TELEPORT_GATEWAY_SELECTOR",
+    "effect": [
+      {
+        "run_eoc_selector": [
+          "EOC_TELEPORT_GATEWAY_SELECTOR_01",
+          "EOC_TELEPORT_GATEWAY_SELECTOR_02",
+          "EOC_TELEPORT_GATEWAY_SELECTOR_03",
+          "EOC_TELEPORT_GATEWAY_SELECTOR_04",
+          "EOC_TELEPORT_GATEWAY_SELECTOR_05",
+          "EOC_TELEPORT_GATEWAY_SELECTOR_06",
+          "EOC_TELEPORT_GATEWAY_SELECTOR_07",
+          "EOC_TELEPORT_GATEWAY_SELECTOR_08",
+          "EOC_TELEPORT_GATEWAY_SELECTOR_09",
+          "EOC_TELEPORT_GATEWAY_SELECTOR_10"
```

Comment:
Is this maximum of 10 memorizations an in-universe limit, or is this just what's currently practically to implement with EOCs (and in-universe, a sufficiently intelligent and skilled TP wielding a relevant crystal could achieve more)? I would like to know for some fiction I'm failing to write.